service/service.go

```diff
@@ -102,9 +118,19 @@ func New(config Config) (*Service, error) {
 		LogsToken:  config.Viper.GetString(config.Flag.Service.Tenant.Ignition.Debug.LogsToken),
 	}
 
+	oidcClientId := config.Viper.GetString(config.Flag.Service.Installation.Tenant.Kubernetes.API.Auth.Provider.OIDC.ClientID)
+	if oidcClientId == "" {
+		oidcClientId = config.Viper.GetString(config.Flag.Service.Azure.ClientID)
+	}
+
+	oidcIssuerUrl := config.Viper.GetString(config.Flag.Service.Installation.Tenant.Kubernetes.API.Auth.Provider.OIDC.IssuerURL)
+	if oidcIssuerUrl == "" {
+		oidcIssuerUrl = fmt.Sprintf("https://login.microsoftonline.com/%s/v2.0", config.Viper.GetString(config.Flag.Service.Azure.TenantID))
+	}
```

Comment:
This is the part I'm less sure about. I'm configuring OIDC using the same clientid and tenantid than the operator is using. That's how we do it during tests. Does it make sense?

Reply:
Yeah we can do that, i mean we are not authenticating with OIDC anyway

Reply:
I mean we are not testing here that we authenticate but  that the params are passed correctly right?

Reply:
Yes. We don't change anything if params are passed, which is the case in our installations. We just change how it works when params are not passed. In that case I use the same credentials than the operator is using.